src/BrandedNavBar/NavBar.tsx

```diff
@@ -83,19 +70,40 @@ const MediumNavBar = ({
 
 const pixelDigitsFrom = (pixelString) => parseInt(pixelString, 10);
 
-const SelectNavBarBasedOnWidth = ({ width, defaultOpen, breakpointUpper, ...props }: any) => {
+const SelectNavBarBasedOnWidth = ({
+  width,
+  defaultOpen,
+  breakpointUpper,
+  brandingLinkHref = "/",
+  brandingLinkTo,
+  brandingLinkComponent,
+  logoSrc,
```

Comment:
should we change the `logoSrc` to match what the components are using `showNulogyLogo`

Reply:
@haideralsh Currently, `logoSrc` when is specified, we show the `logoSrc` logo in the upper-left and a Nulogy logo in the upper-right. When `logoSrc` isn't specified, we show a Nulogy logo only in the upper-left. I didn't want to change the existing behaviour of the `BrandedNavBar`, so I didn't expose the `showNulogyLogo` prop (which actually only controls if the Nulogy logo in the upper-right shows, so maybe it should be called something else?)